plugin/trino-hive/src/test/java/io/trino/plugin/hive/metastore/thrift/InMemoryThriftMetastore.java

```diff
@@ -205,7 +207,9 @@ public synchronized void createTable(HiveIdentity identity, Table table)
         }
 
         PrincipalPrivilegeSet privileges = table.getPrivileges();
-        if (privileges != null) {
+        if (privileges != null
```

Comment:
drop this change? is it needed?

Reply:
Removed since this is not required for this change